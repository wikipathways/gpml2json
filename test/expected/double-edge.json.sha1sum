27265b49443fa02498311c1b9cb8448f33fb7255 ?-
