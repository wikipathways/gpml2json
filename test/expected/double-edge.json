{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["f9392","db851","dc334","daa35","c07f6","bf413","a5baf","ed50c","f9f79","d13b7","edd26","da897","fe6b5","ab407","afb28","f4e60","eb8a3","f4e10","c7675","e7556","f2036","b4a2a","f32af","b3b3e","ec4f0","d4458"],"drawAs":"rect","gpmlElementName":"Pathway","height":726.4197530864194,"kaavioType":"Group","name":"New Pathway","padding":5,"type":["Pathway"],"width":1103.0114676388162,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20171009","standardName":"New Pathway","displayName":"New Pathway","textContent":"Name: New Pathway","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=New%20Pathway&doSearch=1","id":"hash://sha256/82fc601ff9ec192a8dd518be52d462bb741b7703eabd16b3a22d3535b075df11"},"entityMap":{"f9392":{"id":"f9392","borderStyle":"double","x":94.03622943362623,"y":115.48922423233859,"width":65.08503401360538,"height":65.08503401360531,"zIndex":28690,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"db851":{"id":"db851","borderStyle":"double","x":168.8661614064152,"y":115.48922423233859,"width":208.82653061224403,"height":65.08503401360531,"zIndex":28691,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"dc334":{"id":"dc334","borderStyle":"double","x":94.03622943362623,"y":192.15589089900442,"width":65.08503401360538,"height":178.8945578231295,"zIndex":28692,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"daa35":{"id":"daa35","borderStyle":"double","x":510.4730543141706,"y":27.581750993876085,"width":64.08503401360538,"height":64.08503401360531,"zIndex":28693,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c07f6":{"id":"c07f6","borderStyle":"double","x":585.3029862869694,"y":27.581750993876085,"width":207.82653061224403,"height":64.08503401360531,"zIndex":28694,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"bf413":{"id":"bf413","borderStyle":"double","x":510.4730543141706,"y":104.24841766053808,"width":64.08503401360538,"height":177.8945578231295,"zIndex":28695,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"a5baf":{"id":"a5baf","borderStyle":"double","x":17.134654630477364,"y":39.375051003992326,"width":64.08503401360538,"height":64.08503401360531,"zIndex":28696,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ed50c":{"id":"ed50c","borderStyle":"double","x":91.96458660326635,"y":39.375051003992326,"width":207.82653061224403,"height":64.08503401360531,"zIndex":28697,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f9f79":{"id":"f9f79","borderStyle":"double","x":17.134654630477364,"y":116.04171767065827,"width":64.08503401360538,"height":177.8945578231295,"zIndex":28698,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"d13b7":{"id":"d13b7","borderStyle":"double","x":589.9595953639694,"y":105.72239702348602,"width":65.08503401360538,"height":65.08503401360531,"zIndex":28701,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"edd26":{"id":"edd26","borderStyle":"double","x":664.7895273367678,"y":105.72239702348602,"width":208.82653061224403,"height":65.08503401360531,"zIndex":28702,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"da897":{"id":"da897","borderStyle":"double","x":589.9595953639694,"y":182.38906369014757,"width":65.08503401360538,"height":178.8945578231295,"zIndex":28703,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":2,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"fe6b5":{"id":"fe6b5","borderStyle":"double","x":676.7791829928346,"y":192.54198465235154,"width":66.08503401360538,"height":66.08503401360531,"zIndex":28704,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ab407":{"id":"ab407","borderStyle":"double","x":754.7243797631429,"y":192.54198465235154,"width":209.82653061224403,"height":66.08503401360531,"zIndex":28705,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"afb28":{"id":"afb28","borderStyle":"double","x":676.7791829928346,"y":269.20865131901405,"width":66.08503401360538,"height":179.8945578231295,"zIndex":28706,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f4e60":{"id":"f4e60","borderStyle":"double","x":179.13760400407315,"y":202.3088118612042,"width":66.08503401360538,"height":66.08503401360531,"zIndex":28707,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"eb8a3":{"id":"eb8a3","borderStyle":"double","x":256.1823754010034,"y":202.3088118612042,"width":209.82653061224403,"height":66.08503401360531,"zIndex":28708,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f4e10":{"id":"f4e10","borderStyle":"double","x":179.13760400407315,"y":278.97547852787164,"width":66.08503401360538,"height":179.8945578231295,"zIndex":28709,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":3,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c7675":{"id":"c7675","borderStyle":"double","x":264.90879044475196,"y":297.9670039516014,"width":67.08503401360538,"height":67.08503401360531,"zIndex":28710,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"e7556":{"id":"e7556","borderStyle":"double","x":345.38843993166455,"y":297.9670039516014,"width":210.82653061224403,"height":67.08503401360531,"zIndex":28711,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f2036":{"id":"f2036","borderStyle":"double","x":264.90879044475196,"y":380.2833881323913,"width":67.08503401360538,"height":180.8945578231295,"zIndex":28712,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b4a2a":{"id":"b4a2a","borderStyle":"double","x":763.9627988120449,"y":275.48831233596945,"width":67.08503401360538,"height":67.08503401360531,"zIndex":28713,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"f32af":{"id":"f32af","borderStyle":"double","x":845.0232603798652,"y":275.48831233596945,"width":210.82653061224403,"height":67.08503401360531,"zIndex":28714,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"b3b3e":{"id":"b3b3e","borderStyle":"double","x":763.9627988120449,"y":354.97983775969266,"width":67.08503401360538,"height":180.8945578231295,"zIndex":28715,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":4,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"ec4f0":{"id":"ec4f0","borderStyle":"double","x":865.9474944593912,"y":375.1771293194038,"width":69.08503401360538,"height":182.8945578231295,"zIndex":28716,"fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":6,"color":"#999999","rotation":0,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"d4458":{"id":"d4458","borderStyle":"double","x":356.0083866762463,"y":387.9374931122026,"width":69.08503401360538,"height":182.8945578231295,"zIndex":28717,"backgroundColor":"#ffcccc","fontSize":10,"verticalAlign":"middle","drawAs":"rect","borderRadius":15,"borderWidth":6,"color":"#999999","rotation":0,"textAlign":"center","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"}}}
